tests/nat/test_static_nat.py

```diff
@@ -0,0 +1,238 @@
+import copy
+import time
+import json
+
+import pytest
+
+from nat_helpers import DIRECTION_PARAMS
+from nat_helpers import apply_static_nat_config
+from nat_helpers import nat_zones_config
+from nat_helpers import nat_statistics
+from nat_helpers import nat_translations
+from nat_helpers import perform_handshake
+from nat_helpers import get_network_data
+from nat_helpers import generate_and_verify_traffic
+from nat_helpers import get_l4_default_ports
+from tests.common.helpers.assertions import pytest_assert
+
+
+class TestStaticNat(object):
+    """ TestStaticNat class for testing static nat """
+
+    @pytest.mark.nat_static
+    def test_nat_static_basic(self, ptfhost, testbed, duthost, ptfadapter, setup_test_env, protocol_type):
+        interface_type, setup_info = setup_test_env
+        setup_data = copy.deepcopy(setup_info)
+        direction = 'host-tor'
+        nat_type = 'static_nat'
+        # Define network data and L4 ports
+        network_data = get_network_data(ptfadapter, setup_data, direction, interface_type, nat_type=nat_type)
+        src_port, dst_port = get_l4_default_ports(protocol_type)
+        # Set NAT configuration for test
+        apply_static_nat_config(duthost, network_data.public_ip, network_data.private_ip, nat_entry=nat_type)
+        nat_zones_config(duthost, setup_data, interface_type)
+        # Perform TCP handshake from host-tor
+        perform_handshake(ptfhost, setup_data, protocol_type, direction,
+                          network_data.ip_dst, dst_port,
+                          network_data.ip_src, src_port,
+                          network_data.public_ip)
+        # Send bidirectional traffic
+        for path in DIRECTION_PARAMS:
+            generate_and_verify_traffic(duthost, ptfadapter, setup_data, interface_type, path, protocol_type, nat_type=nat_type)
+
+    @pytest.mark.nat_static
+    def test_nat_static_napt(self, ptfhost, testbed, duthost, ptfadapter, setup_test_env, protocol_type):
+        interface_type, setup_info = setup_test_env
+        setup_data = copy.deepcopy(setup_info)
+        direction = 'leaf-tor'
+        nat_type = 'static_napt'
+        # Define network data and L4 ports
+        network_data = get_network_data(ptfadapter, setup_data, direction, interface_type, nat_type=nat_type)
+        src_port, dst_port = get_l4_default_ports(protocol_type)
+        # Set NAT configuration for test
+        apply_static_nat_config(duthost, network_data.public_ip, network_data.private_ip, nat_entry=nat_type,
+                                protocol_type=protocol_type, global_port=dst_port, local_port=src_port)
+        nat_zones_config(duthost, setup_data, interface_type)
```

Comment:
Configured the static NAT and NAT zones using json files only, It's good to use CLI for atleast 1 test case. 
I see function configure_nat_over_cli() is available in nat_helpers.py file, add a similar one for configuring the NAT zone.

Reply:
@msamichelland will add configuration NAT via CLI in next PR